dockers/docker-database/database_config.json.j2

```diff
@@ -130,6 +130,12 @@
             "instance" : {% if include_remote_db %} "remote_redis" {% else %} "redis" {% endif %}
         }
 {% endif %}
+        ,
+        "BMP_STATE_DB" : {
+            "id" : 19,
+            "separator": "|",
+            "instance" : "redis"
```

Comment:
>redis [](http://example.com/codeflow?start=26&length=5)

This is still the single instance. Will you use another dedicated redis instance? It's okay to have multiple redis-server instances inside one database container.

Reply:
updated to use new redis instance as well.

Reply:
Do you mean `redis_bmp` instance?

Reply:
fixed.